packages/calypso-e2e/src/email-client.ts

```diff
@@ -0,0 +1,61 @@
+import config from 'config';
+import MailosaurClient from 'mailosaur';
+import type { Message, Link } from 'mailosaur/lib/models';
+
+/**
+ * Wrapper client around the Mailosaur.io Node.JS client.
+ */
+export class EmailClient {
+	private client: MailosaurClient;
+
+	/**
+	 * Construct and instance of the EmailClient.
+	 */
+	constructor() {
+		this.client = new MailosaurClient( config.get( 'mailosaurAPIKey' ) );
+	}
+
+	/**
+	 * Given an inbox and email address, retrieves the latest email.
+	 *
+	 * @param param0 Keyed parameter object.
+	 * @param {string} param0.inboxId ID of the inbox to look into. Also known as serverId in Mailosaur parlance.
+	 * @param {string} param0.emailAddress Email address of the recipient.
+	 * @returns {Message} Message object returned by Mailosaur client.
+	 */
+	async getLastEmail( {
```

Comment:
There is a level of assumption made with the way the methods in `EmailClient` have been implemented.

In existing Selenium tests, EmailClient calls return a list of messages, but all `Invite` related tests only ever are interested in the [last email](https://github.com/Automattic/wp-calypso/blob/trunk/test/e2e/specs/specs-calypso/wp-invite-users__revoke-spec.js#L57) received by the recipient.

There is one other instance where the EmailClient is used, and that is with the `Signup` spec. I think when the time comes to migrate the `Signup` spec the necessary changes can be made and that we should stick with this assumption for now.